src/chainstate/burn/db/sortdb.rs

```diff
@@ -2312,6 +2299,27 @@ impl SortitionDB {
         Ok(())
     }
 
+    fn validate_and_insert_epochs(
```

Comment:
can you explain that this will panic if the validation fails?

Reply:
Sure -- added in 4fdde14f5